packages/cli/src/commands/lockedgold/list.ts

```diff
@@ -16,11 +17,23 @@ export default class List extends BaseCommand {
 
   async run() {
     const { args } = this.parse(List)
-    cli.action.start('Fetching commitments...')
+    cli.action.start('Fetching commitments and delegates...')
     const lockedGold = await this.kit.contracts.getLockedGold()
     const commitments = await lockedGold.getCommitments(args.account)
+    const delegates = []
+    for (const role of Roles) {
+      delegates.push({
+        role: role,
+        address: await lockedGold.getDelegateFromAccountAndRole(args.account, Roles.indexOf(role)),
```

Comment:
Let's move `Roles` to contractKit, and then make
`lockedGold.getDelegateFromAccountAndRole()` expect a Role, not a role index